flang/lib/Lower/OpenMP.cpp

```diff
@@ -1490,6 +1493,19 @@ bool ClauseProcessor::processDeviceType(
   return false;
 }
 
+bool ClauseProcessor::processDistSchedule(
+    Fortran::lower::StatementContext &stmtCtx, mlir::UnitAttr &scheduleStatic,
+    mlir::Value &chunkSize) const {
+  if (auto *distScheduleClause = findUniqueClause<ClauseTy::DistSchedule>()) {
```

Comment:
Nit: Expand `auto` here and below.

Reply:
The returned type is spelled out in the initialization expression here and all variables obtained from calling `Fortran::semantics::GetExpr` in this file use `auto` like below, so I prefer following the same approach.